src/Aspire.Hosting.Docker/DockerComposeServiceExtensions.cs

```diff
@@ -48,4 +48,58 @@ public static IResourceBuilder<T> PublishAsDockerComposeService<T>(this IResourc
 
         return builder;
     }
+
+    /// <summary>
```

Comment:
@prom3theu5 This is the pattern we want for helm values for the k8s manifest. I am thinking that before we GA either docker compose or the k8s resource types, the underlying model should be more typed so these aren't strings but something like `YamlValue<string>` (or maybe we go specific and say it's composevalue\<T\> and kubevalue\<T\>), that way.

We need to:
 - Allow going from app model to cdk model
 - Preserve the association between the cdk value and the aspire app model somewhere (for later)